src/shared/components/FlameGraph/components/infrastructure/PprofRequestWithoutMaxNodes.ts

```diff
@@ -0,0 +1,28 @@
+import { Field, Message } from 'protobufjs/light';
+
+export class PprofRequestWithoutMaxNodes extends Message<PprofRequestWithoutMaxNodes> {
```

Comment:
Just renamed it to prevent runtime errors (https://github.com/protobufjs/protobuf.js/issues/1408). This class will be removed very soon because it was used for the legacy comparison pages.